src/common/Pagination/index.js

```diff
@@ -4,20 +4,22 @@ import { Container, Button, TextArea, Text} from "./styled";
 
 const Pagination = () => {
 
+    //tmp Data, also disabled state added to 2 buttons for preview
     const currentPage = 7;
     const maxPages = 100;
+    const isMobile = true;
 
     return(
         <Container>
-            <Button disabled>&#60; <Text>First</Text></Button>
-            <Button disabled>&#60; <Text>Previous</Text></Button>
+            <Button disabled>{isMobile ? <>&#60;&#60;</> : <>&#60; <Text>First</Text></>}</Button>
+            <Button disabled>{isMobile ? <>&#60;</> : <>&#60; <Text>Previous</Text></>}</Button>
             <TextArea>
                 <>
                     <Text inTextArea>Page</Text> {currentPage} <Text inTextArea>of</Text> {maxPages} 
                 </>
             </TextArea>
```

Comment:
I think you should rename this component. For example: Page Number or Pages. "Textarea" is a basic tag - an input designed for a large amount of text. This name here can be confusing.

Reply:
done! :)